uitools/common/src/OAuthUserConfigurationManager.cpp

```diff
@@ -0,0 +1,79 @@
+
+/*******************************************************************************
+ *  Copyright 2012-2025 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+#include "OAuthUserConfigurationManager.h"
+
+// Qt headers
+#include <QList>
+
+// Toolkit headers
+#include "AuthenticatorController.h"
+
+using namespace Esri::ArcGISRuntime::Authentication;
+
+namespace Esri::ArcGISRuntime::Toolkit {
+
+/*!
+  \inmodule Esri.ArcGISRuntime.Toolkit
+  \ingroup GenericTools
+  \class Esri::ArcGISRuntime::Toolkit::OAuthUserConfigurationManager
+  \brief A class that manages the available OAuthUserConfiguration instances for the application.
+
+  All added configurations will become owned by the OAuthUserConfigurationManager. Any parent
```

Comment:
Can a little detail be added for what a "configuration" is? Can we hyperlink to the various "configuration" types?

Reply:
Since this is toolkit doc, we can't directly link to class doc and topics from the Maps SDK doc.

There is only one class called `OAuthUserConfiguration` so hopefully that helps with discoverability. I didn't want to repeat all that doc here again since it would be inferior with no links.

Reply:
I must've done it wrong because it was giving me an error but that doesn't.

I'll push that up.